go.sum

```diff
@@ -1,5 +1,6 @@
 github.com/golang/protobuf v1.2.0 h1:P3YflyNX/ehuJFLhxviNdFxQPkGK5cDcApsge1SqnvM=
 github.com/golang/protobuf v1.2.0/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5yJMmIC1U=
+github.com/google/go-github v17.0.0+incompatible h1:N0LgJ1j65A7kfXrZnUDaYCs/Sf4rEjNlfyDHW9dolSY=
```

Comment:
Please delete this line. It was removed in a recent PR.

Reply:
Sorry that I commit changes bit by bit because I was just online and thought you already done reviewing. Should already include all your review. 

Thank you very much for the review, @gmlewis ! :slightly_smiling_face: